doc/macsec/MACsec_hld.md

```diff
@@ -0,0 +1,788 @@
+    "send_sci":{{true|false}} (OPTIONAL)
+    "rekey_period":{{rekey_period}} (OPTIONAL)
+
+; Defines schema for MACsec profile configuration attributes
+key                         = MACSEC_PROFILE:name      ; MACsec profile configuration
+; field                     = value
+priority                    = DIGITS                   ; For Key server election.
+                                                       ; In 0-255 range with 0 being the highest priority
+cipher_suite                = "GCM-AES-128" / "GCM-AES-256" / "GCM-AES-XPN-128" / "GCM-AES-XPN-256"
+                                                       ; The cipher suite for MACsec.
+                                                       ; Default GCM-AES-128 if this filed not exist
+primary_cak                 = 32HEXDIG / 64HEXDIG      ; Primary Connectivity Association Key
+primary_ckn                 = 64HEXDIG                 ; Primary CAK Name
+fallback_cak                = 32HEXDIG / 64HEXDIG      ; Fallback Connectivity Association Key
+fallback_ckn                = 64HEXDIG                 ; Fallback CAK Name
+policy                      = "bypass" / "integrity_only" / "security"
+                                                       ; MACsec policy.
+                                                       ; BYPASS: All traffics will bypass the SecY.
+                                                       ; INTEGRITY_ONLY: All traffics, except EAPOL, will be
+                                                       ; converted to MACsec packets without encryption.
+                                                       ; SECURITY: All traffics, except EAPOL, will be
+                                                       ; encrypted by SecY.
+enable_replay_protect       = "true" / "false"         ; Whether enable replay protect. Default false
+replay_window               = DIGITS                   ; Replay window size that is the number of
+                                                       ; packets that could be out of order. This filed
+                                                       ; works only if ENABLE_REPLAY_PROTECT is true.
+                                                       ; Default 0 which means no replay window and
+                                                       ; strict check. The maximum is 2^32-1
+send_sci                    = "true" / "false"         ; Whether send SCI. Default true
+rekey_period                = DIGITS                   ; The period of proactively refresh (Unit second).
+                                                       ; Default 0 which means never proactive refresh SAK.
+; The profile cannot be deleted if it has been used by a port.
+```
+
+#### 3.1.2 Port Table
+
+``` rfc5234
+PORT|{{port_name}}
+    "macsec":{{profile}} (OPTIONAL)
+
+; Defines schema for MACsec profile configuration attributes
+key                         = PORT:name               ; Interface name
```

Comment:
For PortChannel support, should the physical interface name or the PortChannel name be specified here?

Reply:
physical interface name.